tests/commandnotfound_plugin/commandnotfound.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, cmdfilter
+
+
+class TestCommandNotFoundFilter(BotPlugin):
+    def __init__(self, *args, **kwargs):
+        super(TestCommandNotFoundFilter, self).__init__(*args, **kwargs)
+        self.catch_unprocessed = True
+
+    @cmdfilter
```

Comment:
Maybe adding a parameter to the decorator is better that a boolean to the plugin like: @cmdfilter(not_found=true) ?